### ISSUE TITLE
pip无法安装

### ISSUE THREAD

**Issue by liuxsdev** [LitoMore/fanfou-sdk-python#2]
`pip install fanfou_sdk` 报错:
![image](https://github.com/LitoMore/fanfou-sdk-python/assets/11004456/6b56e056-c513-4c45-991f-7c7abbe2e2ee)
是不是`setup.py`里多了这行 https://github.com/LitoMore/fanfou-sdk-python/blob/96717393a093596dcfe3491055ab18ae18c0002e/setup.py#L4C1-L4C18

**Comment by LitoMore** [comment]
需要先裝好它所需要的其他庫：

```sh
pip install requests
pip install six
pip install fanfou_sdk
```

**Comment by LitoMore** [comment]
哦，`import fanfou_sdk` 在這裡確實可以不需要，我更新一下

**Comment by LitoMore** [comment]
It is fixed in `0.0.3`. Thank you for reporting this!

**Comment by liuxsdev** [comment]
感谢，已经可以正常安装了
